etc/flatpak/remotes.d/flathub.flatpakrepo

```diff
@@ -0,0 +1,8 @@
+[Flatpak Repo]
```

Comment:
Just curious - can the repo be added via adding:

```
flatpak remote-add --system --if-not-exists flathub https://flathub.org/repo/flathub.flatpakrepo
flatpak remote-modify --system flathub --no-filter --title="Flathub (System)"
```

To the Containerfile?

Reply:
Yes to the first command.  
Not tried the second

Reply:
Interesting. Went ahead and gave it a shot, and this ran without failure (the script gets read from my recipe and called in the build script called by the Containerfile): https://github.com/EyeCantCU/BlueWhaleOS/commit/13281120e877f77a47c1c9621fb84aaa43fb0fd0

Throwing this into my Containerfile also seems to have worked:

```
RUN flatpak remote-add --system --if-not-exists flathub https://flathub.org/repo/flathub.flatpakrepo && \
    flatpak remote-modify --system flathub --no-filter --title="Flathub (System)"
```

Either way, it's awesome to hear the remote can at least be added that way

Reply:
I see why manually including this is needed now. My instructions above only work for the build process. The remote never gets added to the users system the way I set it up